skyfield/iokit.py

```diff
@@ -259,7 +259,7 @@ def tle_file(self, url, reload=False, filename=None):
 
         """
         with self.open(url, reload=reload, filename=filename) as f:
-            return list(parse_tle_file(f))
+            return list(parse_tle_file(f, ts=ts))
```

Comment:
Let's switch this to just `(f, ts)`. Adding a keyword argument when a positional would work will make this call more expensive for Python, and the value of the keyword for documentation is not that high here, since the routine being called is right in the same file.